ozkour-front/tests/unit/TestAuthentication.spec.js

```diff
@@ -0,0 +1,80 @@
+import { shallowMount } from "@vue/test-utils";
+import AuthenticationDefault from "@/components/AuthenticationDefault.vue"
+import '@testing-library/jest-dom/extend-expect';
+import {ref} from 'vue'
+import { useAuth0 } from '@auth0/auth0-vue';
+
+jest.mock("@auth0/auth0-vue");
+
+const mockedUseAuth0 = jest.mocked(useAuth0, true);
+
+const user = {
+  email: "user@test.com",
+  email_verified: true,
+  name: "John Doe"
+};
+
+describe("Authentication", () => {
+  describe("Authentication Default Component", () => {
+    it("should display the user name when the user is authenticated", () => {
+      mockedUseAuth0.mockReturnValue({
+        isAuthenticated: ref(true),
+        user: ref(user),
+      });
+      const wrapper = shallowMount (AuthenticationDefault);
+  
+      const userName = wrapper.find('span').text()
+              
+      expect(userName).toBe("John Doe")
+    })
+    it("should display login button when the user isn't authenticated", () => {
```

Comment:
ca me perturbe bcp qu'il n'y a pas de saut de ligne entre les "it". Vous pouvez laisser comme ca mais je tenais quand meme a vous le dire!